doc/zephyr.doxyfile.in

```diff
@@ -1105,7 +1105,11 @@ EXCLUDE_PATTERNS       =
 # wildcard * is used, a substring. Examples: ANamespace, AClass,
 # ANamespace::AClass, ANamespace::*Test
 
-EXCLUDE_SYMBOLS        = _* \
+EXCLUDE_SYMBOLS        = _[^t]* \
+                         _t[^r]* \
+                         _tr[^a]* \
+                         _tra[^c]* \
+                         _trac[^k]* \
```

Comment:
This is really a dirty workaround for the fact that the `_track_list_k_xxx` variables are global and yet prefixed with an `_`
I will probably look into adding proper public macros to tracking.h that expose them more cleanly.

Reply:
Yeah, maybe starting with `_`  was a poor choice to start with (!). I'd drop it (or change to some other "namespace"), but downstream (this is supposed to be used by debugging tools out there) usage would break. Not sure if the macros would be of much help to them - I imagine they simply need to find the right symbol, hiding that under pretty macros could be counterproductive.
In other words, this is not an API to be used from inside Zephyr (I guess one could find usage, but that's beside the point), so it's somewhat special. Maybe they do deserve special treatment?